platforms/Windows/bundle/installer.wxs

```diff
@@ -119,6 +120,15 @@
           <MsiProperty Name="INSTALLREDIST" Value="[OptionsInstallRedistArm64]" />
         </MsiPackage>
       <?endif?>
+
+      <?if $(INCLUDE_ANDROID_SDKS) == true ?>
```

Comment:
We should allow for this to extend to other architectures like we did for Windows.